internal/vm/vm.go

```diff
@@ -55,7 +54,6 @@ func New(opts Opts) (*VM, error) {
 	l.PreloadModule("go.rand", rand.Loader)
 	l.PreloadModule("go.uuid", uuid.Loader)
 	l.PreloadModule("go.ipmatcher", ipmatcher.Loader)
-	l.PreloadModule("go.re2", gluare.Loader)
```

Comment:
We should keep these; no reason to remove them.

Reply:
Leaving this here means I load in two places?
https://github.com/Kong/goks/pull/26/files#diff-52bbf82efe0ff91b7d6fef2ba5c50f71e7ee2b462edb2a74b0424fa40f73a910R26

Reply:
yep, does that cause problems?

Reply:
Not that I can tell. I've made some updates such that we don't reference `go.re2` at all. If you agree with those changes, then I suggest we don't import here.

Reply:
So because we have multiple goks PRs in flight and @tjasko has #27 in flight which uses `go.re2` we should keep it. We can always make another PR to clean up the last little bit of `go.re2` usage in our Lua code, but since it doesn't hurt I think we should keep it; especially since it is not a maintenance burden.